cumulusci/cli/tests/test_config.py

```diff
@@ -0,0 +1,156 @@
+from datetime import date
+from datetime import timedelta
+import mock
+import os
+import sys
+import unittest
+
+import click
+
+import cumulusci
+from ..config import CliConfig
+from cumulusci.core.config import OrgConfig
+from cumulusci.core.exceptions import ConfigError
+from cumulusci.core.exceptions import NotInProject
+from cumulusci.core.exceptions import OrgNotFound
+from cumulusci.core.exceptions import ProjectConfigNotFound
+
+class TestCliConfig(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        os.chdir(os.path.dirname(cumulusci.__file__))
+        os.environ['CUMULUSCI_KEY'] = '1234567890abcdef'
+
+    def test_init(self):
+        config = CliConfig()
+
+        self.assertSetEqual(
+            {'cumulusci', 'tasks', 'flows', 'services', 'orgs', 'project'},
```

Comment:
i think we should probably assert that this is a subset of the keys, the `config` object is sorta like `__dict__`, i don't like asserting about its exact structure